xs/src/libslic3r/Point.cpp

```diff
@@ -18,6 +18,12 @@ Point::operator==(const Point& rhs) const
     return this->coincides_with(rhs);
 }
 
+bool
+Point::operator<(const Point& rhs) const
+{
+    return this->x < rhs.x || this->y < rhs.y;
```

Comment:
Is there an external constraint/property in mind when comparing Points in this manner? Sorting with this seems particularly arbitrary.

My first thought on "is Point X < Point Y" was to compare the length of their distance vectors to some third point (such as the origin).